manifests/0000_30_cluster-api_02_crd.infrastructure-gcp.yaml

```diff
@@ -1283,12 +1289,6 @@ spec:
                           description: Region is the name of the region where the
                             Subnetwork resides.
                           type: string
-                        routeTableId:
-                          description: 'EnableFlowLogs: Whether to enable flow logging
-                            for this subnetwork. If this field is not explicitly set,
-                            it will not appear in get listings. If not set the default
-                            behavior is to disable flow logging.'
-                          type: boolean
```

Comment:
They dropped a field?

Reply:
Renamed it - https://github.com/kubernetes-sigs/cluster-api-provider-gcp/pull/665

But yeah, it breaks backward compatibility

Reply:
Right, and we are making this change between major releases right? So it should be ok. In the future we would need to  be consistent about upgrade paths for this

Reply:
Yes, correct.